src/frontend/src/scheduler/distributed/stage.rs

```diff
@@ -701,23 +724,21 @@ impl StageRunner {
             let query_id = self.stage.query_id.id.clone();
             let stage_id = self.stage.id;
             let task_id = *task;
-            spawn(async move {
-                if let Err(e) = client
-                    .abort(AbortTaskRequest {
-                        task_id: Some(risingwave_pb::batch_plan::TaskId {
-                            query_id: query_id.clone(),
-                            stage_id,
-                            task_id,
-                        }),
-                    })
-                    .await
-                {
-                    error!(
-                        "Abort task failed, task_id: {}, stage_id: {}, query_id: {}, reason: {}",
-                        task_id, stage_id, query_id, e
-                    );
-                };
-            });
+            if let Err(e) = client
```

Comment:
`spawn` is removed but the comments remain unchanged:

```
2. Send RPC to each compute node for each task asynchronously.
```

Reply:
And Why? 👀

Reply:
There is not much reason, and it was removed when debugging an issue.